packages/fx-core/tests/core/core.test.ts

```diff
@@ -283,17 +283,19 @@ describe("Core basic APIs", () => {
           const validSettingsResult = validateSettings(projectSettings);
           assert.isTrue(validSettingsResult === undefined);
 
-          const envInfoResult = await loadSolutionContext(tools, inputs, projectSettings);
-          if (envInfoResult.isErr()) {
-            assert.fail("failed to load env info");
-          }
+          if (!commonTools.isMultiEnvEnabled()) {
```

Comment:
why don't you check `solutionContext` anymore?

Reply:
solutionContext should be in an env, migrate v1 do not have any env.